src/main/java/teammates/common/util/Const.java

```diff
@@ -743,7 +743,8 @@ private Const() {
         public static final String FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHRECIPIENT = "constSumPointsForEachRecipient";
         public static final String FEEDBACK_QUESTION_CONSTSUMDISTRIBUTEUNEVENLY = "constSumUnevenDistribution";
         public static final String FEEDBACK_QUESTION_CONTRIBISNOTSUREALLOWED = "isNotSureAllowedCheck";
-        public static final String FEEDBACK_QUESTION_GENERATEDOPTIONS = "generatedOptions";
+        public static final String FEEDBACK_QUESTION_MCQGENERATEDOPTIONS = "mcqGeneratedOptions";
```

Comment:
As part of the discussion in #8373, we prefer to have `_` after every word to make it readable. As of the rest and consistency issue, we can probably have another code quality issue to fix it.